src/Tribe/Views/V2/Template/Excerpt.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * Handles the manipulation of the excerpt.
+ *
+ * @since   TBD
+ *
+ * @package Tribe\Events\Views\V2\Template
+ */
+
+namespace Tribe\Events\Views\V2\Template;
+
+/**
+ * Class Excerpt
+ *
+ * @since   TBD
+ *
+ * @package Tribe\Events\Views\V2\Template
+ */
+class Excerpt {
+
+	/**
+	 * Excerpt constructor.
+	 *
+	 * @since TBD
+	 */
+	public function __construct() {}
+
+	/**
+	 * Filters the excerpt length.
+	 *
+	 * Set the excerpt length for list and day view.
+	 *
+	 * @since TBD
+	 *
+	 * @param int $length The excerpt length.
+	 *
+	 * @return int The excerpt length modified, if necessary.
+	 */
+	public function maybe_filter_excerpt_length( $length ) {
+		$context = tribe_context();
+		$view = $context->get( 'event_display_mode', 'list' );
+
+		if ( 'list' === $view || 'day' === $view ) {
```

Comment:
use `in_array()` and return 30 directly, avoids the person having to read the whole method to get to the return.

Reply:
I thought it would be better for readability if it was this way.